### PR TITLE
Mention 'Klarna Pay now' where relevant

### PR DESCRIPTION
**Pull request by mvdpanne**
Use the preferred order of:
- Klarna Pay now
- Klarna Pay later
- Klarna Slice it

### CODE REVIEW

File: source/orders/overview.rst [hunk 1]
```diff
@@ -20,8 +20,7 @@ This is mandatory for pay-after-delivery methods. Only shipped amounts will be s
 The following payment methods require the Orders API and cannot be used with the
 :doc:`Payments API </payments/accepting-payments>`:
 
-* *Pay after delivery* payment methods, such as Klarna Pay later and Klarna Slice it
-* Klarna Pay now
+* *Pay after delivery* payment methods, such as Klarna Pay now, Klarna Pay later and Klarna Slice it
```
Comment (mvdpanne) [inline source/orders/overview.rst:23]: I wasn't entirely sure whether Klarna Pay now would be considered a 'pay after delivery' payment method.

In `source/reference/v2/settlements-api/list-settlement-captures.rst`, it is mentioned that:
> Captures are used for *pay after delivery* payment methods.

Which is why I decided to consider it as part of that group for this PR.

Comment (ndijkstra) [reply source/orders/overview.rst:23]: No, Pay Now is not a pay after delivery method, but it is using captures

File: source/reference/v2/settlements-api/list-settlement-captures.rst [hunk 2]
```diff
@@ -15,8 +15,8 @@ List settlement captures
 Retrieve all captures in a certain settlement.
 
 Captures are used for *pay after delivery* payment methods. The only payment methods at the moment that have this flow
```
Comment (ndijkstra) [inline source/reference/v2/settlements-api/list-settlement-captures.rst:17]: *pay after delivery* is mentioned here as well, but not the case for Pay now.

**Review by zoltanczirkos-mollie — APPROVED** [review]

File: source/reference/v2/refunds-api/create-order-refund.rst [hunk 3]
```diff
@@ -14,8 +14,8 @@ Create order refund
 
 When using the Orders API, refunds should be made for a specific order.
 
-When using *pay after delivery* payment methods such as *Klarna Pay later* and *Klarna Slice it*, your customer will
-receive a credit invoice with more information about the refunded products.
+When using *pay after delivery* payment methods such as *Klarna Pay now*, *Klarna Pay later* and *Klarna Slice it*,
+your customer will receive a credit invoice with more information about the refunded products.
```
Comment (mvdpanne) [inline source/reference/v2/refunds-api/create-order-refund.rst:18]: @ndijkstra would the customer receive a credit invoice for Klarna Pay now as well?

Comment (mvdpanne) [reply source/reference/v2/refunds-api/create-order-refund.rst:18]: Does it make sense then to remove this paragraph?

**Pull request merged**